<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LMS_MessageNode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_BranchNode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_EventNode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_EntryNode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_JumpNode</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="subtype_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.type_0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.type_1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.type_2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.type_3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.type_4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.string_table</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.type_6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LMS_NodeSubtypes.none</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="subtype_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtype</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
